recipe/meta.yaml

```diff
@@ -14,38 +14,40 @@ source:
     - cmake.patch  # [win]
 
 build:
-  number: 1
-  skip: true  # [win and vc<14]
+  number: 0
+  skip: True  # [win and vc<14]
   run_exports:
     - {{ pin_subpackage(name, max_pin='x.x') }}
-
-# TODO: conda-forge giflib missing headers on windows?
+  ignore_run_exports:
+    - libwebp  # [osx and arm64]
+    - xz
 
 requirements:
   build:
     - {{ compiler('c') }}
-    - autoconf  # [not win]
-    - automake  # [not win]
-    - make  # [not win]
-    - libtool  # [not win]
-    - m4  # [not win]
-    - pkg-config  # [not win]
-    - cmake  # [win]
-    # latest build of 7.71.1 (build 6) is broken
-    - curl <7.71.1  # [win]
-    - m2-unzip  # [win]
+    - autoconf      # [not win]
+    - automake      # [not win]
+    - make          # [not win]
+    - libtool       # [not win]
+    - m4            # [not win]
+    - pkg-config    # [not win]
+    - cmake         # [win]
+    - curl          # [win]
+    - m2-unzip      # [win]
   host:
-    - giflib  # [not win]
-    - jpeg
-    - libpng
-    - libtiff
-    - libwebp
+    # TODO: conda-forge giflib missing headers on windows?
+    - giflib        # [not win]
+    - jpeg          # [not win]
+    - libpng        # [not win]
+    - libtiff       # [not win]
+    - libwebp       # [not win]
     - openjpeg
     - xz
-    - zlib
+    - zlib          # [not win]
   run:
     # libwebp doesn't have run_exports yet (on version in conda-forge-pinning)
-    - libwebp
+    - libwebp       # [not win]
```

Comment:
should be 
```
- libwebp # [not (win or (osx and arm64))]
```

Reply:
I've left only `libwebp-base  # [osx and arm64]`